### DISCUSSION TITLE
Start Menu and Taskbar Issues

### DISCUSSION THREAD

**Discussion by ASchrader19** [valinet/ExplorerPatcher#3273] [Q&A]
Greetings!

I'm new to this GitHub page, and I'm loving this software, but I'm having some issues with it.
On the start menu, typically there's a settings wheel above the user icons, but on ExplorerPatcher, it doesn't look like there is one, is there a way to enable the settings wheel/icon.

I'm also curious if there's a way to shrink the taskbar text a little bit to not have it spell out the entire program name and such.

Attaching the pictures to explain what my issues are:

![image_2024-05-14_112443642](https://github.com/valinet/ExplorerPatcher/assets/80972327/a53491f2-bd8c-4030-bb2a-318fa5e7714e)

![image_2024-05-14_112550481](https://github.com/valinet/ExplorerPatcher/assets/80972327/c7e9ee76-3f0b-4985-b8cf-bc83013303ae)

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.  If you can, please change it to english first.

Please include any other software you have installed that modifies the UI of windows.

As for shrinking the taskbar, that is not a feature of EP.  All you can do is right click on it and untick lock the taskbar.  Then you can add another row by resizing it.

**Reply by ASchrader19** [reply]
Windows Version and Build: Windows 11 Pro | 22631.3593

I haven't changed any settings in terms of properties on the application; it loaded up without the settings icon on the left-hand side.

Explorer Patcher is the only thing I'm using in terms of modifying the UI of windows.

**Reply by pyrates999** [reply]
Please post explorer patcher version.

**Reply by ASchrader19** [reply]
Patcher version is 22621.3527.65.1

**Reply by pyrates999** [reply]
Please try the latest pre-release here: https://github.com/valinet/ExplorerPatcher/releases/tag/22621.3527.65.5_6f6666b

**Reply by ASchrader19** [reply]
Sadly still no luck on getting the setting wheel on the start menu :(

**Comment by Amrsatrio** [comment]
@ASchrader19 Go to Settings > Personalization > Start > Choose which folders appear on Start. The settings wheel (or other buttons) can be made to show on both 11 and 10 start menus.

*Hmph. Is it that hard to discover? Guess I have to make a FAQ entry for this one*
